cli_script.sh

```diff
@@ -200,23 +200,44 @@ else
 EOF
 fi
 
-for pkg in marathon-lb cassandra kafka zeppelin; do
-    cmd="dcos --log-level=ERROR package install --yes"
-    if [[ $pkg = 'marathon-lb' ]] && ! $DCOS_OSS; then
-        cmd="$cmd --options=options.json"
-    fi
-    cmd="$cmd $pkg"
-    if $USE_STABLE; then
-        key="${pkg^^}_STABLE"
-        key="${key//-/_}" # replace - with _ for marathon-lb
-        eval ver='$'$key
-        cmd="$cmd --package-version=$ver"
-    fi
-    demo_eval "$cmd"
-done
+if $INFRA_ONLY; then
+    for pkg in marathon-lb cassandra kafka; do
+        cmd="dcos --log-level=ERROR package install --yes"
+        if [[ $pkg = 'marathon-lb' ]] && ! $DCOS_OSS; then
+            cmd="$cmd --options=options.json"
+        fi
+        cmd="$cmd $pkg"
+        if $USE_STABLE; then
+            key="${pkg^^}_STABLE"
+            key="${key//-/_}" # replace - with _ for marathon-lb
+            eval ver='$'$key
+            cmd="$cmd --package-version=$ver"
+        fi
+        demo_eval "$cmd"
+    done
```

Comment:
Really not a fan of copy/pasting this function as its doing some wacky stuff. Its pretty easy to rework this bit so we don't do that